src/pages/[teamName].tsx

```diff
@@ -681,7 +682,7 @@ export default withError<GET_SERVER_SIDE_PROPS_ERROR, ITeamProps>(
                       }
                       backgroundColor="transparent"
                       icon={<FallbackIcon color={`mode.${colorMode}.icon`} />}
-                      name={session.user.name}
+                      name=""
```

Comment:
```suggestion
                      name={session.user.name`'s uploaded avatar`}
```

What about adding that it is an avatar?